demos/weather_forecast/run_demo.sh

```diff
@@ -18,22 +41,26 @@ start_demo() {
     echo ".env file created with OPENAI_API_KEY."
   fi
 
-  # Step 3: Start Arch
+  # Step 4: Start Arch
   echo "Starting Arch with arch_config.yaml..."
   archgw up arch_config.yaml
 
-  # Step 4: Start Network Agent
-  echo "Starting Network Agent using Docker Compose..."
-  docker compose up -d  # Run in detached mode
+  # Step 5: Start Network Agent with the chosen Docker Compose file
+  echo "Starting Network Agent using $COMPOSE_FILE..."
+  docker compose -f "$COMPOSE_FILE" up -d  # Run in detached mode
 }
 
 # Function to stop the demo
 stop_demo() {
-  # Step 1: Stop Docker Compose services
-  echo "Stopping Network Agent using Docker Compose..."
-  docker compose down
+  echo "Stopping all Docker Compose services..."
+
+  # Stop all services by iterating through all configurations
+  for compose_file in ./docker-compose*.yaml; do
```

Comment:
Why are we looping through these files? shouldn't we always have one compose file to shut down?

Reply:
Different compose files would shut down different things, so this was just a way to sort of ensure everything is shutdown

Reply:
wait are you recommending having a compose file specific to shutdown?

Reply:
you only start up one docker-compose file. So why are you shutting down all of them?

Reply:
because when sh run_demo.sh down is called, there is no way to know which docker compose file was used

Reply:
Ask for a parameter, else shutdown the default compose file. This would be consistent with the start command. Agree we can do better here, but I won't block on that.